pom.xml

```
@@ -23,7 +23,7 @@
   </parent>
 
   <artifactId>google-kubernetes-engine</artifactId>
-  <version>0.8.10-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
```

```suggestion
  <version>${revision}.${changelist}</version>
```
if we are going to keep `0.8.${#commitsOnMager}.v${gitSha}`